<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>771</width>
     <height>351</height>
    </rect>
   </property>
   <property name="plainText">
    <string>这是一段文字描述
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_light">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>light</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dark">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>dark</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
